WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsHostingController.swift

```diff
@@ -1,12 +1,15 @@
 import SwiftUI
 
 final class PaymentMethodsHostingController: UIHostingController<HostedPaymentMethodsView> {
+    weak var parentController: UIViewController?
+
     init(viewModel: PaymentMethodsViewModel) {
         super.init(rootView: HostedPaymentMethodsView(viewModel: viewModel))
 
         // Needed because a `SwiftUI` cannot be dismissed when being presented by a UIHostingController
         rootView.dismiss = { [weak self] in
             self?.dismiss(animated: true, completion: nil)
```

Comment:
I don't think this is needed anymore! 
I tested all the cases (cash, card, payment link) without this line and dismiss works as expected.

Reply:
Makes sense since we're calling the dismiss method in the parent view controller. Thanks for catching this!